### PR TITLE
check authentication before request

### PR DESCRIPTION
**Pull request by jgsqware**
Signed-off-by: Julien Garcia Gonzalez <julien@giantswarm.io>

### CODE REVIEW

File: cli.py [hunk 1]
```diff
@@ -69,6 +76,12 @@ def remute_channel(channel_id):
         sys.stderr.write("Please set the SLACK_TOKEN environment variable.\n")
```
Comment (marians) [inline cli.py:76]: Can we make it this?

    if TOKEN is None or TOKEN == "":

**Review by marians — APPROVED** [review]
LGTM and works for me!

But, I would test for the case of the empty string, so we don't need the round trip.

File: cli.py (hunk 1 above)
Comment (jgsqware) [reply cli.py:76]: Done

**Review by marians — APPROVED** [review]

**Comment by marians** [comment]
Cool!

**Pull request merged**